### ISSUE TITLE
TensorRT Wrong Option: Unknown provider option: "trt_force_timing_cache".

### ISSUE THREAD

**Issue by NiiightmareXD** [pykeio/ort#100]
Hi I tried using the TensorRT using default option but it fails:
code:
```rs
    let model = include_bytes!("../yolov8n-seg.onnx");

    let environment = Environment::builder()
        .with_name("yolov8n-seg")
        .with_log_level(LoggingLevel::Warning)
        .with_execution_providers([ExecutionProvider::TensorRT(
            TensorRTExecutionProviderOptions::default(),
        )])
        .build()
        .unwrap()
        .into_arc();

    let model = SessionBuilder::new(&environment)
        .unwrap()
        .with_execution_providers([ExecutionProvider::TensorRT(
            TensorRTExecutionProviderOptions::default(),
        )])
        .unwrap()
        .with_parallel_execution(true)
        .unwrap()
        .with_memory_pattern(true)
        .unwrap()
        .with_model_from_memory(model)
        .unwrap();
```
error:
```
WARN ort::execution_providers: An error occurred when attempting to register `TensorrtExecutionProvider`: D:\a\_work\1\s\onnxruntime\core\providers\tensorrt\tensorrt_execution_provider_info.cc:98 onnxruntime::TensorrtExecutionProviderInfo::FromProviderOptions [ONNXRuntimeError] : 1 : FAIL : provider_options_utils.h:148 onnxruntime::ProviderOptionsParser::Parse Unknown provider option: "trt_force_timing_cache". 
```

**Comment by decahedron1** [comment]
Fixed in [1.16.2](https://crates.io/crates/ort/1.16.2)

**Comment by NiiightmareXD** [comment]
thanks for the release
